package.json

```diff
@@ -32,7 +33,7 @@
   "dependencies": {
     "es6-collections": "0.x",
     "skatejs-named-slots": "1.x",
-    "webcomponents.js": "webcomponents/webcomponentsjs#b77ca74"
```

Comment:
Oh really? So when you say "our v1 branch", do you mean https://github.com/skatejs/webcomponentsjs/tree/v1 ? I also noticed https://github.com/skatejs/webcomponentsjs/tree/skate-v1 . Either way, I'd be very happy to stick to not fork it like this.